admin/includes/languages/english/coupon_restrict.php

```diff
@@ -41,7 +41,7 @@
 define('TEXT_ALL_MANUFACTURERS_REMOVE', 'Remove All Manufacturer Products');
 
 define('HEADER_PRODUCT_STATUS', 'Status');
-define('HEADER_PRODUCT_MODEL', 'Model');
+define('HEADER_PRODUCT_MODEL', TABLE_HEADING_MODEL);
```

Comment:
Does this actually work correctly right now? Doesn't the Admin currently load this file before english.php where that constant is defined?

Reply:
Ugh you're right.  I'll put the string back in here, and then do a new PR to make these defines more like all the rest (TABLE_HEADING_PRODUCTS_MODEL instead of HEADER_PRODUCT_MODEL, etc.)